### PR TITLE
Add clarification about IP forwarding requirement (bsc#1150212)

### PR DESCRIPTION
**Pull request by r0ckarong**

### CODE REVIEW

File: adoc/deployment-sysreqs.adoc [hunk 1]
```diff
@@ -203,6 +203,25 @@ The management workstation needs at least the following networking permissions:
 
 |===
 
+==== Port Forwarding
```
Comment (aojea) [inline adoc/deployment-sysreqs.adoc]: s/Port/IP/

**Comment by aojea** [comment]
@ereslibre @flavio PTAL

**Comment by aojea** [comment]
@r0ckarong we should rename the title to "IP forwarding" instead of "Port forwarding" :sweat_smile: 
it sounds good to me

File: adoc/deployment-sysreqs.adoc [hunk 2]
```diff
@@ -203,6 +203,25 @@ The management workstation needs at least the following networking permissions:
...
+
+The link:https://kubernetes.io/docs/concepts/cluster-administration/networking/[{kube} networking model] requires that your nodes have IP forwarding enabled in the kernel.
+`skuba` checks this value when installing your cluster and installs a rule in `/etc/sysctl.d/90-skuba-net-ipv4-ip-forward.conf` to make it persistent.
+
+Other software can potentially install rules with higher priority overriding this value and causing the cluster to fail.
```
Comment (ereslibre) [inline adoc/deployment-sysreqs.adoc]:
```suggestion
Other software can potentially install rules with higher priority overriding this value and causing machines to not behave as expected after rebooting.
```

File: adoc/deployment-sysreqs.adoc [hunk 3]
```diff
@@ -203,6 +203,25 @@ The management workstation needs at least the following networking permissions:
...
+[source,bash]
+----
+# sysctl net.ipv4.ip_forward
+
+net.ipv4.ip_forward = 1
+----
+
+`net.ipv4.ip_forward` must be set to `1`.
+
+
```
Comment (ereslibre) [inline adoc/deployment-sysreqs.adoc]: Additionally, you can check in what order persisted rules are processed by running `sysctl --system -a`.

**Pull request merged**